app/page.tsx

```diff
@@ -12,7 +12,6 @@ const Home = () => {
           заедница од ентузијасти и професионалци. Со LearnHub, вие не сте само дел од заедница; вие
           градите иднина."
       />
-      <BlogList />
```

Comment:
why did you remove this?

Reply:
Oh it was causing conflicts in the begining... i believe because the final update of that branch wasnt pushed and the page wasnt rendering.When i merged the main with my branch everything was fixed with the latest updates of the other branches.But works fine.